### ISSUE TITLE
send apprise alert

### ISSUE THREAD

**Issue by pczhang** [louislam/uptime-kuma#4928]
### ⚠️ Please verify that this question has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Hello,

I am trying to set up apprise notification on uptime kuma. When I do 
```
apprise --body="Test Message" apprises://host/config_id/?tags=all
```

I can receive the notification.
However, when I put `apprises://host/config_id/?tags=all` to uptime kuma and click test, I got the error.
would you please let me know what I did wrong. thank you!

### 📝 Error Message(s) or Log
```
date	stream	content
2024/07/10 11:46:07	stderr	}
2024/07/10 11:46:07	stderr	  stderr: ''
2024/07/10 11:46:07	stderr	    '2024-07-10 11:46:07,023 - ERROR - There are no service(s) to notify\n', 
2024/07/10 11:46:07	stderr	    '2024-07-10 11:46:07,023 - ERROR - Could not load Apprise API URL: apprises://a...e/2...9/?tags=all\n' + 
2024/07/10 11:46:07	stderr	  stdout: '2024-07-10 11:46:07,023 - WARNING - The Apprise API token specified (27c180dd723b67931458f95c6b4c7726e7e98d2f6657475b97546c77958da249) is invalid.\n' + 
2024/07/10 11:46:07	stderr	  signal: null, 
2024/07/10 11:46:07	stderr	  code: 1, 
2024/07/10 11:46:07	stderr	    at ChildProcess._handle.onexit (node:internal/child_process:303:5) {
2024/07/10 11:46:07	stderr	    at maybeClose (node:internal/child_process:1098:16)
2024/07/10 11:46:07	stderr	    at ChildProcess.emit (node:events:517:28)
2024/07/10 11:46:07	stderr	    at ChildProcess.done (/app/node_modules/promisify-child-process/index.cjs:70:19)
2024/07/10 11:46:07	stderr	Error: Process exited with code 1
```

### 🐻 Uptime-Kuma Version

1.23.13

### 💻 Operating System and Arch

Synology NAS

### 🌐 Browser

Brave

### 🖥️ Deployment Environment

- Runtime: Docker version 20.10.23, build 876964a 
- Database: N/A
- Filesystem used to store the database on: N/A
- number of monitors: 3

**Comment by pczhang** [comment]
i am certain that is the correct one. and using the same token "apprise --body="Test Message" apprises://host/token/?tags=all" can send message.

**Comment by pczhang** [comment]
btw: in apprise API page, they call it "config ID", but you call it token. They mean to be the same thing, right?

**Comment by CommanderStorm** [comment]
Cannot confirm. What page did you look at?
https://github.com/caronc/apprise/wiki/Notify_apprise_api does only specify tokens, no ids

**Comment by pczhang** [comment]
![Screenshot-2](https://github.com/louislam/uptime-kuma/assets/416650/deb9b1f1-b149-4043-b907-4152b16a1d1e)
I referred to the apprise setup page. they called it "config id". I assume this is the token in the wiki link you sent.

**Comment by CommanderStorm** [comment]
I don't see how to get the needed parameter from the docs or from https://hub.docker.com/r/caronc/apprise.
=> Please reopen an issue with https://github.com/caronc/apprise-api instead. I have no idea how this software works and digging through that code is not very productive at 4:00 am

**Comment by louislam** [comment]
### TL;DR;

Define your apprise config name by `https://host/{custom config id}`.

`{custom config id}`'s length must be 32 or below.

For example:
http://unraid:8000/cfg/12345678901234567890123456789012

### Reasons

This should be an upstream issue. The issue had been fixed in Apprise CLI 1.8.0, but Uptime Kuma's docker image came with Apprise CLI 1.6.0.

https://github.com/caronc/apprise/releases/tag/v1.8.0
> Update Apprise API (apprise//) token length by @isometimescode in https://github.com/caronc/apprise/pull/1120

Before Apprise CLI 1.8.0, it accepts token length < 32 only.

Also, before Apprise API 0.9.6, the config id is always defined by custom input, but now the config id is auto-generated with length 64, which length size cannot fit into any older versions of Apprise CLI.

https://github.com/caronc/apprise-api/releases/tag/v0.9.6
> New Configuration Generator button added to make it easier to create obfuscated (and non-guessable) unique configuration keys.

My suggestion to Apprise API would be changing the auto-generated key length to 32 instead of 64 for better backward compatibility 

cc: @caronc (in case you are not aware of this)

**Comment by pczhang** [comment]
Thanks, @louislam ! That was exactly the problem. Changing to 32-length token solves the issue.
